src/metabase/notification/payload/impl/system_event.clj

```diff
@@ -108,9 +110,9 @@
                                       :after 24.99}
                              :status {:before "active"
                                       :after "on sale"}}}
-     [:map-of :keyword [:map
-                        [:before :any]
-                        [:after :any]]]]]])
+     [:map-of :string [:map
```

Comment:
Can we rather map the keys back to keywords? String keys feel like an easy gotcha when handling them.